<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="database.integration.service.UserServiceIT" tests="3" skipped="0" failures="0" errors="0" timestamp="2024-04-05T16:23:48" hostname="LOCALHOST" time="0.01">
  <properties/>
  <testcase name="delete()" classname="database.integration.service.UserServiceIT" time="0.004"/>
  <testcase name="save()" classname="database.integration.service.UserServiceIT" time="0.002"/>
  <testcase name="findLogin()" classname="database.integration.service.UserServiceIT" time="0.004"/>
  <system-out><![CDATA[Hibernate: 
    select
        u1_0.id,
        u1_0.birthday,
        u1_0.image,
        u1_0.name,
        u1_0.password,
        u1_0.status 
    from
        users u1_0 
    where
        u1_0.id=?
2024-04-05T19:23:48.801+03:00  INFO 14360 --- [    Test worker] by.service.UserService                   : Attempt to delete User object by your id, in method delete()
2024-04-05T19:23:48.802+03:00  INFO 14360 --- [    Test worker] by.service.UserService                   : Attempt to extract UserDto object in method findById()
Hibernate: 
    select
        u1_0.id,
        u1_0.birthday,
        u1_0.image,
        u1_0.name,
        u1_0.password,
        u1_0.status 
    from
        users u1_0 
    where
        u1_0.password=?
2024-04-05T19:23:48.808+03:00  INFO 14360 --- [    Test worker] by.service.UserService                   : The User with name Imir was login
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
